api/catalog/api/serializers/audio_serializers.py

```diff
@@ -171,6 +173,15 @@ class AudioSerializer(MediaSerializer):
         help_text="A link to an endpoint that provides similar audio files.",
     )
 
+    # Add-on data
+    peaks = serializers.SerializerMethodField()
```

Comment:
Should we name this something less specific than `peaks` considering in the future other things could potentially live in that response?

Reply:
This method field is specifically obtaining the audio peaks using the `get_peaks` function. It's not serializing the AddOn object.

Reply:
Sorry! I realized this later and forgot to remove this comment.